linters/i18n.js

```diff
@@ -9,7 +9,7 @@ const htmllint = require("htmllint");
 const Reporter = require("../reporter");
 const TranslationFilterScanner = require("./translation_filter_scanner");
 
-const PLURALIZATION_KEYS = ["zero", "one", "two", "other"];
+const PLURALIZATION_KEYS = ['zero', 'one', 'two', 'few', 'many', 'other'];
```

Comment:
I can confirm that this is the complete list present in [CLDR](https://github.com/unicode-org/cldr).